tutorials-and-examples/inference-servers/checkpoints/README.md

```diff
@@ -15,4 +15,8 @@ Now you can use it in a [Kubernetes job](../jetstream/maxtext/single-host-infere
 - -i=INFERENCE_SERVER
 - -b=BUCKET_NAME
 - -m=MODEL_PATH
+- -q=QUANTIZE
```

Comment:
qq, does this also apply to the Maxtext case?

Reply:
This does not apply to the MaxText case because currently there is no direction for creating a quantized checkpoint for MaxText

Reply:
readme here is shared by both MaxText/Pytorch-xla right?

Reply:
as discussed, we may need to mention which args are for MaxText, which ones are for Pytorch/XLA